core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.aarch64.txt

```diff
@@ -32,15 +32,17 @@ container-selinux
 #cri-tools-1.13.0
 curl
 dejavu-sans-fonts # for grafana
-docker-ce-19.03.14
-docker-ce-cli-19.03.14
+docker-ce-20.10.8
+docker-ce-cli-20.10.8
+docker-ce-rootless-extras-20.10.8
 ebtables
 elasticsearch-oss-7.10.2 # for opendistroforelasticsearch & logging roles
 ethtool
 filebeat-7.9.2
 firewalld
 fontconfig # for grafana
 fping
+fuse3-libs # for docker-ce-rootless-extras
```

Comment:
Required to avoid this error:

```
Error: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)
           Requires: libfuse3.so.3(FUSE_3.2)(64bit)
Error: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)
           Requires: libfuse3.so.3(FUSE_3.0)(64bit)
Error: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)
           Requires: libfuse3.so.3()(64bit)
```